File: 05 SWEA/D1/Java/src/Prob2025.java

/**
* title  : 2025. N줄덧셈 (D1)
* time   : 104ms
* memory : 16076kb
*/

import java.io.*;
 
public class Prob2025 {
    
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        
        int N = Integer.parseInt(br.readLine());
        int answer = 0;

        if (N % 2 == 0) {
            answer = (1 + N) * (N / 2);
        } else {
            answer = N* ((N - 1) / 2) + N;
        }

        System.out.println(answer);
    }
}

File: 05 SWEA/D1/Python/Prob2025.py

"""
title  : 2025. N줄덧셈 (D1)
time   : 114ms
memory : 43760kb
"""

def solution():

    N = int(input())

    if (N % 2):
        print(N * (N//2) + N)
    else:
        print((1 + N) * (N // 2))

solution()